src/e2e/app/cli/sqlite/tests/tests/rest_api/test_odf_core.rs

```diff
@@ -13,12 +13,15 @@ use kamu_cli_e2e_common::prelude::*;
 
 kamu_cli_run_api_server_e2e_test!(
     storage = sqlite,
-    fixture = kamu_cli_e2e_repo_tests::test_rest_api_request_dataset_tail,
-    // We need synthetic time for the tests, but the third-party JWT code
-    // uses the current time. Assuming that the token lifetime is 24 hours, we will
-    // use the projected date (the current day) as a workaround.
-    options = Options::default().with_today_as_frozen_system_time(),
-    extra_test_groups = "engine, datafusion"
+    fixture = kamu_cli_e2e_repo_tests::rest_api::test_odf_info_st,
```

Comment:
Comment for readers:

Note: git[hub] got a little confused incorrectly displaying the difference -- it's not a moved and updated file, just a new one.